INN/umbrella-ipbs
# Fonts, colors, typography styles, block color palette

## Changes

This pull request makes the following changes:

- Adds fonts from designs
- Adds colors from design
- Updates typography styling to match designs
- Adds block color palette using the colors added

## Why



For #62

## Testing/Questions

Features that this PR affects:

- Styling site wide


Questions that need to be answered before merging:

- [ ] I tried to not add colors that were too similar to other ones. Am I missing any that should be added?
- [ ] Any other styles we should add to this PR?

Steps to test this PR:

1. View frontend and verify all fonts, colors, and typography matches the designs so far
2. Add a block and make sure the block color palette works as expected
